pkg/cmd/edge_functions/delete/delete.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/MakeNowJust/heredoc"
```

Comment:
I think this file will conflict with mine, because I've deleted both the legacy list and delete commands in my PRs